samples/video-decode-display/index.html

```diff
@@ -1,30 +1,77 @@
-<!doctype html>
 <!DOCTYPE html>
 <html>
 <head>
-  <title>WebCodec MP4 frame extration demo</title>
-  <meta http-equiv="origin-trial" content="ArdlZia9G23wi6S2x/vVoTla5x9r1wtreSPqhUq36tpGH7HRmAkPgpBnpkfePFeClDJDzzYAhtDPoI5hJioArAYAAABjeyJvcmlnaW4iOiJodHRwczovL3czYy5naXRodWIuaW86NDQzIiwiZmVhdHVyZSI6IldlYkNvZGVjcyIsImV4cGlyeSI6MTYzODQwMzE5OSwiaXNTdWJkb21haW4iOnRydWV9" />
+  <title>WebCodec MP4 decode sample</title>
 </head>
 <body>
   <p>
-    This demo extracts all frames from an MP4 file and renders them to a canvas as fast as possible. It uses <a href="https://github.com/gpac/mp4box.js/">mp4box.js</a> to parse and demux the file.
+    This demo decodes all frames from an MP4 file and renders them to a canvas as fast as possible.
+    It uses <a href="https://github.com/gpac/mp4box.js/">mp4box.js</a> for demuxing.
   </p>
-  <canvas></canvas>
-</body>
 
-<script type="module">
-  var demuxDecodeWorker;
+  <p>
+    Renderer:
+    <label for="renderer_2d">
+      <input id="renderer_2d" type="radio" name="renderer" value="2d"> 2D
+    </label>
+    <label for="renderer_webgl">
+      <input id="renderer_webgl" type="radio" name="renderer" value="webgl" checked> WebGL
+    </label>
+    <label for="renderer_webgl2">
+      <input id="renderer_webgl2" type="radio" name="renderer" value="webgl2"> WebGL 2
+    </label>
+    <label for="renderer_webgpu">
+      <input id="renderer_webgpu" type="radio" name="renderer" value="webgpu"> WebGPU
```

Comment:
I'd put some disclaimer note that this requires a chrome flag or similar (and may generally be broken) while WebGPU is still being developed.

Reply:
Done.